packages/openid4vc/src/openid4vc-issuer/OpenId4VcIssuerService.ts

```diff
@@ -514,7 +519,7 @@ export class OpenId4VcIssuerService {
       })
 
       deferredCredentialResponse = vcIssuer.createDeferredCredentialResponse({
-        credentials: credentials.credentials,
+        credentials: credentials.credentials.map((c) => ({ credential: c })),
```

Comment:
ℹ️ For deferred issuance, we supported Draft 15, so we just use the latest format, as this is a fix.